.github/workflows/rebuild-100k.yml

```
@@ -0,0 +1,57 @@
+name: Rebuild 100k sample
+
+on:
+  # cron job once a week on mondays
+  schedule:
+    - cron:  '* * * * 1'
+  # Manually triggered using GitHub's UI
+  workflow_dispatch:
+
+jobs:
+  gisaid:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
```

```suggestion
    - uses: actions/checkout@v3
```

v2 is deprecated soon

---

Done :)